plugins/woocommerce/client/admin/client/settings-payments/components/payment-settings-section/payment-settings-section.tsx

```diff
@@ -0,0 +1,24 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+
+/**
+ * Internal dependencies
+ */
+import './payment-settings-section.scss';
+
+export const PaymentSettingsSection: React.FunctionComponent< {
```

Comment:
`React.FunctionComponent` (or the shorthand React.FC) is not needed. It is even [discouraged](https://github.com/facebook/create-react-app/pull/8177) for React 17 or TypeScript lower than 5.1.